README.md

```diff
@@ -108,10 +108,16 @@ foo(
 )
 ```
 
-Prettier bans all custom styling by parsing it away and re-printing
+Prettier bans all custom styling\* by parsing it away and re-printing
 the parsed AST with its own rules that take the maximum line width
 into account, wrapping code when necessary.
 
+_\*Well actually, some original styling is preserved when practical—see
+[empty lines] and [multi-line objects]._
```

Comment:
It might be helpful to have the asterisks link to this footnote, so that readers can easily jump to it. One way to do that is to change this line to

```markdown
 <a name="styling-footnote">_\*Well actually, some original styling is preserved when practical—see
  [empty lines] and [multi-line objects]._</a>
```

and change each asterisk to:

```markdown
[\*](#styling-footnote)
```